hack/e2e-apps/run-kubernetes.sh

```
@@ -4,6 +4,7 @@ run_kubernetes_test() {
     local port="$3"
     local k8s_version=$(yq "$version_expr" packages/apps/kubernetes/files/versions.yaml)
 
+  kubectl -n tenant-test delete kuberneteses.apps.cozystack.io $name --ignore-not-found
```

Shellcheck (0.10.0)

[warning] name is referenced but not assigned.

(SC2154)